README.md

```diff
@@ -364,20 +351,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                      didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
 
         // Override point for customization after application launch.
-
-        Theme.themeColor = UIColor.white
-        Theme.tintColor = UIColor(red: 3 / 255, green: 124 / 255, blue: 1, alpha: 1) //blue
-        Theme.Button.color = Theme.tintColor
-        Theme.Icon.color = Theme.tintColor
         Theme.Icon.backgroundColor = UIColor(red: 219 / 255, green: 241 / 255, blue: 1, alpha: 1) // Light blue
```

Comment:
Theme.Icon does not have backgroundColor.

Reply:
You yourself have changed it ``Theme.Icon.primaryBackgroundColor` | `0xE5F7FA` | The icon primary background color |` in the Theme class.

Reply:
> Theme.Icon does not have backgroundColor.

Removed backgroundColor

Reply:
> You yourself have changed it ``Theme.Icon.primaryBackgroundColor`|`0xE5F7FA` | The icon primary background color |` in the Theme class.

Since primaryBackgroundColor exist in the Theme class , it was added in Readme file